src/core/InformationPanel/InformationPanel.tsx

```diff
@@ -0,0 +1,166 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { IconButton } from '../Buttons';
+import { CommonProps, useMergedRefs, useTheme } from '../utils';
+import SvgCloseSmall from '@itwin/itwinui-icons-react/cjs/icons/CloseSmall';
+import '@itwin/itwinui-css/css/information-panel.css';
+
+export type InformationPanelProps = {
+  /**
+   * Label shown at the top of the panel.
+   */
+  label?: React.ReactNode;
+  /**
+   * Is the panel open?
+   * @default false
+   */
+  isOpen?: boolean;
+  /**
+   * Orientation of the panel.
+   * @default 'vertical'
+   */
+  orientation?: 'vertical' | 'horizontal';
+  /**
+   * Callback fired when close icon is clicked.
+   */
+  onClose?: () => void;
+  /**
+   * Custom header action buttons shown alongside the close button.
+   * @example
+   * [
+   *   <IconButton styleType='borderless' onClick={() => {}}><SvgEdit /></IconButton>,
+   *   <IconButton styleType='borderless' onClick={() => {}}><SvgDelete /></IconButton>,
+   * ]
+   */
+  headerActions?: React.ReactNode[];
```

Comment:
We might have the same problem like in `Header` when users wants custom stuff that we haven't thought of. What if we have a prop for header and have a sub-component `InfomartaionPanelHeader`?

Reply:
I don't think we should have mixed sub-component namings right now.

Reply:
so you would also want me to create `SidenavSubmenuHeader`? weird naming